src/main/java/io/github/seggan/sfcalc/CalcExecutor.java

```diff
@@ -85,14 +84,22 @@ public boolean onCommand(CommandSender sender, Command command, String s, String
             amount = 1;
         } else {
             if (PatternUtils.NUMERIC.matcher(args[1]).matches()) {
-                amount = Long.parseLong(args[1]);
+                try {
+                    amount = Long.parseLong(args[1]);
+                    if (amount == 0) {
+                        sender.sendMessage(plugin.invalidNumberString);
+                        return true;
+                    }
+                } catch (NumberFormatException e) {
+                    sender.sendMessage(plugin.invalidNumberString);
+                    return true;
+                }
```

Comment:
Use the regex pattern check instead of try-catch

Reply:
I cant figure out this matcher thing at all :(

Reply:
look at existing code

Reply:
The numeric matcher will not work for this, I need to check if the value is greater than long max value not whether it's a number

Reply:
ok then